debbuild

```diff
@@ -1574,8 +1722,24 @@ sub splitver {
      $req .= $ver if $ver;
   }
   # Pick up the details of versioned build requirements
-  my ($pkg,$rel,$ver) = $req =~ /([\w.+-]+)\s*([><=]+)\s*([\w:.~+-]+)/;
+  my ($pkg,$rel,$ver) = $req =~ /([\w.+\-()]+)\s*([><=]+)\s*([\w:.~+-]+)/;
+  $pkg =~ s/\(([^\)]+)\)/-$1/g;
   $rel =~ s/^([><])$/$1$1/;
+
+  # We need this workaround as _ is foridden in the package name
+  #  and probably was changed to - on building such packages.
+  $pkg =~ tr/_/-/;
```

Comment:
This conversion requires a warning printed out so the user knows they have an invalid name specification and that we're correcting for it.

Reply:
Didn't find any warning for such conversation in original source:
https://github.com/debbuild/debbuild/blob/master/debbuild#L703-L706

Reply:
Hmm, fair.